fms_files.cmake

```diff
@@ -1,4 +1,5 @@
 list(APPEND fms_src_files
+    FMS/affinity/fms_affinity.F90
```

Comment:
Why are we adding this file back in? We took it out in the past because it broke the macOS builds (and possibly other stuff, can't remember). @DusanJovic-NOAA

Reply:
Also, the executable permissions should be remove again (github shows 644->755)

Reply:
@DomHeinzeller  executable permissions changed

Reply:
@jiandewang this didn't work as expected. You change `CMakeLists.txt` frorm 0644 to 0755 instead of `fms_files.cmake` back from 0755 to 0644. Please look at https://github.com/ufs-community/ufs-weather-model/pull/290/files and remove the executable bit `x` for **all** files that your PR added it to (six files in total).

<img width="803" alt="Screen Shot 2020-12-03 at 5 18 01 AM" src="https://user-images.githubusercontent.com/8006981/101017166-f6c3f980-3526-11eb-876b-dff04065bbc1.png">